php/one/App/swoole.php

```diff
@@ -7,6 +7,11 @@
 
 define('_APP_PATH_VIEW_', __DIR__ . '/View');
 
+function uuid()
```

Comment:
what is this function for ?

Reply:
By default, the framework sets a unique id for logging. There is no need for logging here. I rewritten it.